### ISSUE TITLE
'NoneType' object has no attribute 'get'

### ISSUE THREAD

**Issue by acarl005** [cevoaustralia/aws-google-auth#52]
```
$ aws-google-auth -I C01sopdwp -S 135372135477 -u "$MY_EMAIL" -R us-east-1 -d 3600 -p terraform_state -a

Traceback (most recent call last):
  File "/usr/local/bin/aws-google-auth", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/site-packages/aws_google_auth/__init__.py", line 53, in main
    cli(sys.argv[1:])
  File "/usr/local/lib/python2.7/site-packages/aws_google_auth/__init__.py", line 152, in cli
    google_client.do_login()
  File "/usr/local/lib/python2.7/site-packages/aws_google_auth/google.py", line 136, in do_login
    sess = self.handle_prompt(sess)
  File "/usr/local/lib/python2.7/site-packages/aws_google_auth/google.py", line 244, in handle_prompt
    data_key = response_page.find('div', {'data-api-key': True}).get('data-api-key')
AttributeError: 'NoneType' object has no attribute 'get'
```

Not sure whats happening. I get this after entering my password and doing the MFA challenge from google prompt.

**Comment by acarl005** [comment]
Well this would explain it! I printed the HTML `response_page` and opened it in the browser and this is what I got...

<img width="514" alt="screen shot 2018-02-01 at 4 25 02 pm" src="https://user-images.githubusercontent.com/8334252/35710503-aade65ae-076c-11e8-9595-a200e204e2c7.png">

**Comment by nonspecialist** [comment]
whoops! looks like we shouldn't be assuming that we actually get a valid page back ...

**Comment by stevemac007** [comment]
Would be awesome if we could detect an error page, and pull out the error message to provide contextual feedback to the user.

**Comment by stevemac007** [comment]
Released as part of `0.0.22`
